edi-module/component_group_reader.bal

```diff
@@ -0,0 +1,64 @@
+class ComponentGroupReader {
+
+    SubcomponentGroupReader subcomponentGroupReader = new();
+
+    function read(string compositeText, EDIMapping mapping, EDIFieldMapping emap)
```

Comment:
`mapping` and `emap` names are bit confusing. So can we use `ediMapping` and `ediFieldMapping` instead to avoid ambiguity?